app/models/preservation_policy.rb

```diff
@@ -9,6 +9,11 @@ class PreservationPolicy < ApplicationRecord
   validates :archive_ttl, presence: true, numericality: { only_integer: true, greater_than: 0 }
   validates :fixity_ttl, presence: true, numericality: { only_integer: true, greater_than: 0 }
 
+  # this is a *very* naive cache eviction algorithm: if any pres policy changes, clear
+  # the cache.  we expect pres policies to change very infrequently, so probably no big deal.
```

Comment:
why did you decide to roll your own rather than use Rails built-in caching? is there something about it that won't work for us?

Reply:
@tingulfsen:  my understanding from everything i found online and from @ndushay's assessment is that the rails caching is very focused on caching http responses.  the one library i found that leveraged the rails caching infrastructure for general caching didn't seem very well maintained.

i'm not sure that the rails caching machinery would result in less code, or more readable code, though i do think that if we find ourselves doing more caching, we should more thoroughly explore off the shelf patterns and/or gems, or at least run our code by someone w/ more ActiveRecord experience, to tell us if we're walking into some anti-pattern.

also, if i totally misunderstood rails caching, or missed a library that we could've used, i'm totally open to suggestions.

but i basically took the general approach you used in #351, and added cache eviction, and just cached the id instead of the whole object.  the discussion on that one seemed to indicate that we weren't necessarily itching for a heavier caching solution (or maybe i was misinterpreting that).

Reply:
my feeling was that since the solution is short and relatively readable and hopefully correct and performant for our usage, it felt like "good enough"/"shameless green".